### PR TITLE
version bump: react-ui calendar

### PR DESCRIPTION
**Pull request by cwklausing**
Updated the [react-basic-datepicker](https://github.com/rentpath/react-basic-datepicker) in response to below card. Requires a version bump. 

[Card](https://rentpath.leankit.com/card/697016601)

### CODE REVIEW

File: packages/react-ui-core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rentpath/react-ui-core",
-  "version": "15.2.1",
+  "version": "15.2.2",
```
Comment (tadjohnston) [inline packages/react-ui-core/package.json]: This doesn't bump the datepicker. You'll need to add the datepicker commitish on this line: https://github.com/rentpath/react-ui/pull/383/files#diff-381a8d04c2e36ba268c0723c3a9ce2c5L56

**Comment by tadjohnston** [comment]
Please use `git cz` to commit.

File: packages/react-ui-core/package.json (hunk 1 above)
Comment (cwklausing) [reply packages/react-ui-core/package.json]: @tadjohnston That's the part I was confused about--it's just a reference to the github repo, so how to I show that it's a new version?

Comment (cwklausing) [reply packages/react-ui-core/package.json]: 👍 updated

Comment (tadjohnston) [reply packages/react-ui-core/package.json]: You still don't need this line 😄 . Publishing does this automatically for you.

Comment (cwklausing) [reply packages/react-ui-core/package.json]: ...oh. Gotcha. ha

**Pull request merged**